api/Dockerfile

```diff
@@ -1,7 +1,7 @@
 # NOTE:
 # This Dockerfile is for local development only!
 
-FROM python:3.8-buster
+FROM python:3.8.12-buster
```

Comment:
Why are we pinning a minor version?

Reply:
Because poetry.lock specifies that the Python version must be `>=3.8.6`, but `3.8-buster` has a lower minor version, so the local Docker doesn't work unless we specify the only available minor version >= .6

Reply:
Ah this is for local docker. We'd lose higher minor updates by pinning this but for local dev it's low risk.